CHANGELOG.md

```diff
@@ -122,6 +124,55 @@ the APIs in these libraries are stable, and are ready for production use.
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat(pubsub): implement per-batch compression ([#9270](https://github.com/googleapis/google-cloud-cpp/pull/9270))
+* feat(common): include status error payloads in the RPC log ([#9186](https://github.com/googleapis/google-cloud-cpp/pull/9186))
+
+### [Spanner](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/spanner/README.md)
+
+* feat(common): include status error payloads in the RPC log ([#9186](https://github.com/googleapis/google-cloud-cpp/pull/9186))
+
+### [Storage](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/storage/README.md)
+
+* doc: broken link in GCS README file ([#9264](https://github.com/googleapis/google-cloud-cpp/pull/9264))
+* fix(storage): respect policies in deprecated constructor ([#9255](https://github.com/googleapis/google-cloud-cpp/pull/9255))
+* fix: handle transient failures in the GCE credentials ([#9252](https://github.com/googleapis/google-cloud-cpp/pull/9252))
+* fix: relax parsing for GCE informational fields ([#9251](https://github.com/googleapis/google-cloud-cpp/pull/9251))
+* feat(storage): per-operation options for `Client`
+  ([#9220](https://github.com/googleapis/google-cloud-cpp/pull/9220))
+  ([#9214](https://github.com/googleapis/google-cloud-cpp/pull/9214))
+  ([#9209](https://github.com/googleapis/google-cloud-cpp/pull/9209))
+  ([#9208](https://github.com/googleapis/google-cloud-cpp/pull/9208))
+  ([#9207](https://github.com/googleapis/google-cloud-cpp/pull/9207))
+  ([#9206](https://github.com/googleapis/google-cloud-cpp/pull/9206))
+  ([#9205](https://github.com/googleapis/google-cloud-cpp/pull/9205))
+  ([#9204](https://github.com/googleapis/google-cloud-cpp/pull/9204))
+  ([#9203](https://github.com/googleapis/google-cloud-cpp/pull/9203))
+  ([#9202](https://github.com/googleapis/google-cloud-cpp/pull/9202))
+  ([#9199](https://github.com/googleapis/google-cloud-cpp/pull/9199))
+  ([#9247](https://github.com/googleapis/google-cloud-cpp/pull/9247))
+* fix(storage): do not duplicate debugging headers ([#9162](https://github.com/googleapis/google-cloud-cpp/pull/9162))
+* fix(storage): disable MSVC C++17 deprecation warnings ([#9143](https://github.com/googleapis/google-cloud-cpp/pull/9143))
+* doc(storage): missing options for several functions ([#9146](https://github.com/googleapis/google-cloud-cpp/pull/9146))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* doc: broken link in GCS README file ([#9264](https://github.com/googleapis/google-cloud-cpp/pull/9264))
+* fix: handle transient failures in the GCE credentials ([#9252](https://github.com/googleapis/google-cloud-cpp/pull/9252))
+* fix: relax parsing for GCE informational fields ([#9251](https://github.com/googleapis/google-cloud-cpp/pull/9251))
```

Comment:
these show up in storage and in common. I think we just want storage?

Reply:
Done